pages/admin/dashboard/tools.tsx

```diff
@@ -11,9 +11,26 @@ import {
 import { ToolCard, Modal, AdminLayout } from "@components";
 import { Page } from "types/Page";
 import { useRouter } from "next/router";
+import { isAuthenticated } from "src/utils/auth/authHelpers";
+import { GetServerSideProps } from "next";
 import axios from "axios";
 import useSWR from "swr";
 
+export const getServerSideProps: GetServerSideProps = async ({ req, res }) => {
+    if (process.env.NODE_ENV == "production") {
+        const authProps = await isAuthenticated(req, res, "/redirect", true);
```

Comment:
Can we add a // TODO: change redirect to login page (once we have a login page that's deployed)